
## About this project

As a part of the International Human Epigenome Consortium (IHEC) project, 
we collected ChIP-seq data of histone modifications as well as RNA-seq and DNA methylation data from various vascular cell types from multiple donors. To catalogue and characterize the transcriptomic and epigenomic landscape in the vascular system, we implemented comprehensive analysis for genome-wide data of active histone modifications and gene expression for nine types of human endothelial cells show below.

- Human aortic endothelial cells (HAoECs)
- Human coronary artery endothelial cells (HCoAECs)
- Human endocardial cells (HENDCs)
- Human pulmonary artery endothelial cells (HPAECs)
- Human umbilical vein endothelial cells (HUVECs)
- Human umbilical artery endothelial cells (HUAECs)
- Human common carotid artery endothelial cells (HCCaECs)
- Human renal artery endothelial cells (HRAECs)
- Human great saphenous vein endothelial cells (HGSVECs)

## Data
- Raw read file (Fastq)
   - ChIP-seq: [GEO GSE131953](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE131953)
   - RNA-seq: [GEO GSE131681](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE131681)

- Mapped read file (BAM)
   - [ChIP-seq](https://drive.google.com/open?id=1uz_tX9eue_PLR5AjElYwf46Zv9xxxpsB)
       - Build hg19 and hg38, mapped by [BWA](http://bio-bwa.sourceforge.net/)
   - [RNA-seq](https://drive.google.com/open?id=1XmdM3HQS0-Bto6a-tEdnueoTVXPOeayd)
       - Build hg19 and hg38, mapped by [STAR](https://github.com/alexdobin/STAR)

- Quality check (QC) results
   - [ChIP-seq](https://drive.google.com/open?id=16ialQRmdq-gN6z0_0uyLHNIusK4q0zD-)
   - [RNA-seq](https://drive.google.com/open?id=19ILbnLD1g-GAvXlZBdXGlxyWTe1pNbF4)

- Peak files
   - [The reference promoter sites](https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE131953&format=file&file=GSE131953%5FEC%5Fref%5Fpromoter%2Ebed%2Egz)
   - [The reference enhancer sites](https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE131953&format=file&file=GSE131953%5FEC%5Fref%5Fenhancer%2Ebed%2Egz)
   - The peak list of each sample is also available at [GEO GSE131953](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE131953).

- [Read distribution (PDF)](https://drive.google.com/open?id=16ckH3_HioOUBFMPZvrFAr2i74ApBPrRf)
   - Build hg19, generated by [DROMPA3](https://github.com/rnakato/DROMPA3)
   - **1Mbp-width** is recommended for a general purpose. **100kbp-width** is higher resolution but the file size is large.
   - In **H3K4me3andH3K27ac**, H3K4me3 and H3K27ac are colored in green and orange, respectively.

- [Gene expression data](https://drive.google.com/open?id=1GuG_SGwYbGTbDs7gNjtwZuNW1CRxT6mu)
   - Build GRCh37, generated by [kallisto](https://pachterlab.github.io/kallisto/), [RSEM](https://github.com/deweylab/RSEM) and [edgeR](https://www.bioconductor.org/packages/release/bioc/html/edgeR.html)
   - IMR90 cells from the [SRA SRR2952390](https://www.ncbi.nlm.nih.gov/sra/?term=SRR2952390).

- [ChIA-PET file](https://drive.google.com/open?id=1-7d68UHHt8BFUBLJ-nPk3HPIfoSFIAmF)
    - Fastq data is from [GEO GSE41553](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE41553)
    - Generated by [Mango](https://github.com/dphansti/mango) with default parameter settings.

## Program
- [DROMPA3](https://github.com/rnakato/DROMPA3)
- [SSP](https://github.com/rnakato/SSP)

## Reference
- Nakato R, Wada Y, *et al.*, [Comprehensive epigenome characterization reveals diverse transcriptional regulation across human vascular endothelial cells](https://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-019-0319-0), **Epigenetics & Chromatin**, vol. 12, issue 1, 77, 2019.
- (In Japanese) 中戸隆一郎,　和田洋一郎, 白髭克彦, エピゲノムのダイナミズムを解き明かす大規模比較解析　―血管内皮細胞を例に, **羊土社「実験医学」増刊号** 2016年6月, P1562-1568.

## Contact

This site is maintained by Ryuichiro Nakato (rnakato AT iam.u-tokyo.ac.jp)
